test/multiarch_utils.go

```diff
@@ -60,21 +60,31 @@ func getTestArch() string {
 
 // initImageNames returns the map with arch dependent image names for e2e tests
 func initImageNames() map[int]string {
-	if getTestArch() == "s390x" {
+	switch {
```

Comment:
nit - we could drop the duplication of `getTestArch` here:

```go
switch getTestArch() {
  case "s390x":
  case "ppc64le":
  default:
}
```

Similar comment for the other switch statements in this PR.

Reply:
Made the suggested changes.